resources/popup.html

```diff
@@ -39,7 +40,7 @@
                 </a>
             </div>
             <div class="col s6">
-                <a class="btn waves-effect lime waves-light tooltipped" href="../analytics/index.html" target="_blank"
+                <a id="analytics-link" class="btn waves-effect lime waves-light tooltipped" target="_blank"
```

Comment:
As you can see in the diff, the target was already set to "_blank" before this PR :P
My comment is that in `options.html` and `popup.html`, `target='_blank'` can be removed, as the target is set in `main.js`.